pwa-devdocs/src/technologies/basic-concepts/internationalization/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Internationalization Framework
```

Comment:
I'd be cautious calling this a "framework" we didn't do much framework work and it makes it sound like we invented something, when in fact we just implemented `react-intl`.

Reply:
Updated so now it refers to it as a feature